src/index.ts

```diff
@@ -309,6 +316,12 @@ class Spanner extends GrpcService {
         }
       }
     }
+
+    let interceptors: any[] = [];
+    if (options) {
+      interceptors = options.interceptors || [];
+    }
```

Comment:
Can we also remove this for now, as it is not being used?

Reply:
Done, thanks!